### PR TITLE
wrap QtWebkit in try/except as is an optional library and might not b…

### PR DESCRIPTION
**Pull request by hdd**
Wrap QtWebKit in QtWebKitWidgets to match Qt5

### CODE REVIEW

**Comment by fredrikaverpil** [comment]
This is awesome, thank you @hdd ! 😃 

@mottosso this looks good to me.

**Comment by fredrikaverpil** [comment]
@hdd do you think you could also provide tests in [tests.py](https://github.com/mottosso/Qt.py/blob/master/tests.py)?

**Comment by hdd** [comment]
Sure thing I can, This was just a "early" pull request to show you where I was going.
I'll try to put in place some tests for this.

**Comment by hdd** [comment]
Hope you don't mind, as part of this ticket I improved a bit the setup.py so now you can run:

```
python setup.py nosetests
```

File: setup.py [hunk 1]
```diff
@@ -34,4 +34,5 @@
     zip_safe=False,
     py_modules=["Qt"],
     classifiers=classifiers,
+    setup_requires=['nose>=1.0']
```
Comment (mottosso) [inline setup.py]: This line warrants its own PR, documentation and tests I think. Could we remove this for now, and implement it fully later on?

File: tests.py [hunk 2]
```diff
@@ -188,3 +188,15 @@ def test_sip_api_already_set():
             import sip
             sip.setapi("QString", 1)
             assert_raises(ImportError, __import__, "Qt")
+
+
+def test_qtwebkit_pyside():
```
Comment (mottosso) [inline tests.py]: These tests aren't testing anything of value but rather adds bulk to the project. If we want to test that members of Qt.py are mapped (or not) we should probably do so all together in a single test function.

**Comment by mottosso** [comment]
> Hope you don't mind, as part of this ticket I improved a bit the setup.py so now you can run:

That's allright, but it can be tricky to keep conversations straight when more than one feature is being added.

Great with an example of the end result, but what does `python setup.py nosetests` do? Haven't used it before.

**Comment by hdd** [comment]
I'll be removing it .
Added in the rush of simplifying the testing but better keep it for a separate ticket.
Overall though having nosetests in setup py allows you to wrap flags and features so you don't have to remember them.

File: setup.py (hunk 1 above)
Comment (hdd) [inline setup.py]: it'll be removed, in case we'll be opening a ticket to improve the setup.py

File: tests.py (hunk 2 above)
Comment (hdd) [inline tests.py]: Usually is better tests separated from each other (one test , one assert) , but of course there are exceptions.

Ops, noticed now, half of the test went away before the commit.

**Comment by mottosso** [comment]
> Overall though having nosetests in setup py allows you to wrap flags and features so you don't have to remember them.

I like that, at the moment, these flags are documented in the Dockerfile, but having them in setup.py is more discoverable.

Let's open up a separate PR for it, and make sure to remember nosepipe, --verbose, --with-process-isolation and --with-doctest as well.

File: tests.py [hunk 3]
```diff
@@ -188,3 +188,15 @@ def test_sip_api_already_set():
             import sip
             sip.setapi("QString", 1)
             assert_raises(ImportError, __import__, "Qt")
+
+
+def test_QtWebKitWidgets_available_in_pyside():
```
Comment (mottosso) [inline tests.py]: Let's remove these tests, they aren't doing any good.

We could start testing with runnable examples, but I'm uncertain about what that would achieve also.

**Comment by mottosso** [comment]
This looks good, apart from that one inline comment!

**Pull request merged**